server/src/main/java/org/elasticsearch/action/admin/indices/resolve/ResolveIndexAction.java

```diff
@@ -598,12 +598,12 @@ private static void mergeResults(
 
         private static void enrichIndexAbstraction(
```

Comment:
This API doesn't support yet the failure store, so we will work on it on a subsequent PR